workspaces/adoption-insights/plugins/adoption-insights/src/translations/fr.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createTranslationMessages } from '@backstage/core-plugin-api/alpha';
+import { adoptionInsightsTranslationRef } from './ref';
+
+/**
+ * French translation for Adoption Insights.
+ * @public
+ */
+const adoptionInsightsTranslationFr = createTranslationMessages({
+  ref: adoptionInsightsTranslationRef,
+  messages: {
+    'page.title': "Insights d'Adoption",
+    'header.title': "Insights d'Adoption",
```

Comment:
"Insights" is not a French word. I guess this is the translation for "Adoption Insights". To check with PMs and/or Docs, but perhaps something like this would make more sense? Or leave the branded name "Adoption Insights" as is?

```suggestion
    'page.title': "Statistiques d'adoption",
    'header.title': "Statistiques d'adoption",
```